pkgs/shells/pdksh/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchurl }:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+
+  name = "pdksh-${version}";
+  version = "5.2.14";
+
+  src = fetchurl {
+    url = "http://www.cs.mun.ca/~michael/pdksh/files/${name}.tar.gz";
+    sha256 = "19mzkfmj3ms8sp9m8f6fyba0am6fxnr538xw6b3fv4gja3fvq5db";
+  };
+
+  buildInputs = [ ];
+
+  # Traditionally, PDKSH is used as "pdksh" rather than "ksh"
+  configureFlags = [ "--program-prefix=pd" ];
+
+  patches = [
```

Comment:
I'd very much prefer to have those patch files downloaded from the official site (like our `bash` build and many other packages do it). The Nixpkgs repository is intended for collecting build expressions, and we'd rather not include patches unless it's absolutely necessary. This seems to be no such case, because the patches are readily available elsewhere.